### ISSUE TITLE
Support Paseto v3/v4

### ISSUE THREAD

**Issue by not-my-profile** [sjudson/paseto.js#38]
Paseto v1/v2 is planned to be deprecated on 2022-01-01. See https://github.com/paseto-standard/paseto-spec/pull/13.

**Comment by sjudson** [comment]
Yep, I'm aware. Should hopefully have a modernization together by then.
